.azure-pipelines/markers_check/markers_check.py

```diff
@@ -0,0 +1,56 @@
+import re
+import os
+import sys
+import logging
+
+from natsort import natsorted
+
+
+def collect_all_scripts():
+    '''
+    This function collects all test scripts under the folder 'tests/'
+    and get the topology type marked in the script
+    '''
+    location = sys.argv[1]
+
+    # Recursively find all files starting with "test_" and ending with ".py"
+    # Note: The full path and name of files are stored in a list named "files"
+    files = []
+    for root, dirs, file in os.walk(location):
+        for f in file:
```

Comment:
Variable name `file` could conflict with the `file` builtin module. 
Although it works here, but please do avoid such practice to avoid inadvertent bug in the future.

Reply:
Changed all names related to `file` to `script`